### ISSUE TITLE
$139.50 cash out requested on the 19th of May; request not even acknowledged, let alone fulfilled, more than 7 weeks later

### ISSUE THREAD

**Issue by thesquash** [bountysource/core#1591]
Over the past year, I earned $155 from three open bounties, all on the MATE Desktop project:

 - https://app.bountysource.com/issues/32817997-feature-request-sticky-center-property-for-panel-items/
 - https://app.bountysource.com/issues/41100594-please-add-an-option-to-set-the-max-volume-above-100/
 - https://app.bountysource.com/issues/37931764-clock-doesn-t-get-updated-when-opening-notebook-display-lid/

On the 19th of May (admittedly a Friday), I finally decided I needed my money and submitted a cash out request for all $155 I had accumulated.  I knew BountySource was going to skim 10% off that figure and that I would only get $139.50, but I was willing to accept that as just a nuisance.  I also expected that, since I submitted the claim on a Friday, that BountySource's "volunteers" would not get around to processing the claim for an entire week -- they claim to generally only process claims on Friday (assuming their FAQs are up-to-date).

In reality, though, I've been waiting for seven weeks now (week seven just passed on the 7th of July), with no response:  No money, no rejection, no closure, not even a mere acknowledgement of anything:

![Screenshot at 2023-07-08 20-38-35](https://github.com/bountysource/core/assets/66928007/45cec2a2-c70c-47cf-a9d7-9e34153287bc)

On the 13th of June, I sent BountySource a very polite support request email asking for a status update on my cash out request.  The email's verbatim contents follows:

[quote]
Dear Sir or Madam,

On Friday, May 19 (2023), I requested to cash out my whole BountySource credit,
amounting to $155.00 (with BountySource's cash-out fee factored in, I
understand that I actually would be getting $139.50).  Since that date, three-
and-a-half weeks have passed, while I would have generally expected the
cash-out to be complete within a week or two from my initial request.  Could
you please tell me what is taking the extra time, and if nothing is, please do
what you can to make sure my request is fulfilled in a timely manner?  I am
depending on that money to be available as soon as possible.

I appreciate anything and everything you can do for me regarding this issue.
Thank you very much in advance.

Regards,
Gordon
[/quote]

I promptly received a cryptic automated reply to my email, in broken English, but no human ever replied despite waiting weeks:

[quote]
Your request (859) has successfully been received by our Bountysource support team. We get back to you within the next 48 hours.

To add some comments, please respond to that e-mail.
[/quote]

About a week later, I went BountySource's home page at bountysource.com, and used the "Get in touch" feedback form at the bottom of that page to send another message, this time explaining that their support request system was lousy and that I wanted my money, but had gotten not even a reply from a human after so long.  I would quote my exact words here, but unfortunately I wasn't smart enough  to copy them down before I hit the Send button that time.

Then on the 28th of June, I replied to my original BountySource email and sent this message, hoping someone would notice:

[quote]
Okay, it's been nearly a full six weeks now since I asked for the last cash-out,
and I have received neither my money nor any support reply whatsoever.  You
are disregarding your customers and violating better business practices.

I demand my money right now.
[/quote]

I still have not heard any reply.

---

This is absurd.  Small developers are trying to make ends meet by monetizing real work that they put into open-source projects, and all they get is -- well, I hate to say it, but fiat "money" that they can't cash out for real money to keep a roof over their heads.  They can only use it to post bounties on other open issues, so that another developer can get a bunch of these tokens that they can't use for anything useful.

In other words, BountySource takes people's real money, and then everybody is forever after stuck transacting in this pseudo-economy of value-devoid (presumably Blockchain-based) tokens as "payment" for doing real work.  BountySource rakes in the douch, and developers fail to be compensated for their time and effort.  And if you complain, they won't snub you off; they simply won't reply.

And who are these "volunteers" as mentioned in the FAQs?  I don't see any "Help Wanted" page that lists a job for anybody outside the company, so these people must be inside BountySource.  And BountySource gets a 10% cut from each payout, which must add up to millions of dollars by now, so how these people inside BountySource can be considered volunteers is beyond me.  If the cash out processors aren't getting paid, who else is collecting all that money?  Unless there *are* no volunteers processing cash outs, and all that money is going into a shell account on some Caribbean paradise island nation, or lining the pockets of some high-level official in the Blockchain Group.

I guess you can't really expect much else from a company called the Blockchain Group, can you?

---

It seems like everybody who tries to cash out is getting ignored just like me (#1586, for example).  I will give BountySource a chance to be defended here, though:  If somebody **did** succeed in getting a cash out, please post here.  That way, we can see how many people BountySource is actually serving.  If I get a hundred replies, then BountySource is probably just congested with cash out requests.  If I get nothing, they're probably up to embezzlement, and I'd suggest bounty posters (users who put their money into getting issues solved) do what they can to get their money back and that everybody on BountySource leave the platform.  I know I'm leaving once I have an answer to what's going on here, even if I can't get my cash.

BountySource, show yourselves to be reasonable people, stop harming your reputation, and just pay people their money!

**Comment by Py-Swift** [comment]
![IMG_2665](https://github.com/bountysource/core/assets/107263226/d7a67d0f-0204-4044-a620-cafb0d7a311d)

**Comment by quickwritereader** [comment]
#1586

**Comment by quickwritereader** [comment]
> ![IMG_2665](https://user-images.githubusercontent.com/107263226/253062373-d7a67d0f-0204-4044-a620-cafb0d7a311d.jpeg)

#1586

**Comment by jaapmarcus** [comment]
Same here...
<img width="1240" alt="Screenshot 2023-07-17 at 11 16 40" src="https://github.com/bountysource/core/assets/9754650/5689a3d3-d003-47ae-af2f-6e8f6ef52506">

**Comment by quickwritereader** [comment]
> Same here... <img alt="Screenshot 2023-07-17 at 11 16 40" width="1240" src="https://user-images.githubusercontent.com/9754650/253898914-5689a3d3-d003-47ae-af2f-6e8f6ef52506.png">

@abebeos #1586

**Comment by quickwritereader** [comment]
any updates?

**Comment by thesquash** [comment]
If you're asking me if there's any updates on my end, then sort of.  After waiting 8 weeks with no response whatsoever, I cancelled the cash-out request and opened a new one, otherwise identical to the old one:

![Screenshot 2023-07-26 at 15-01-18 Bountysource](https://github.com/bountysource/core/assets/66928007/b042339f-bd0e-469e-980d-e3a5a199e8c2)

My rationale was that maybe BountySource would take notice of a brand new cash out request, and they were shelving stale old ones for when they were finished processing the new ones.  Looking back on it, I don't really see any logic in that hypothesis, but hey, I wanted to give them a chance.

But guess what -- still no reply.  Nothing.  Not even a peep.  I really wonder, if you go to their physical office in San Francisco, if there's even any people there.  Heh -- maybe they boarded up shop due to all the crime in San Fran and are too afraid to go back to work.  Still, you'd think remote work -- *ahem* -- remote *volunteer* work would be an option.  It's certainly no excuse to stop paying everybody!  And even so, we deserve an explanation and a little warning at minimum!

By the way, I haven't yet heard a peep from anybody who successfully **got** their money.  Does that mean that people who get their money don't have any reason to check the issue tracker and see that there's a payout problem, and reply to this issue (likely)?  Or does it mean that nobody's receiving real money right now?

**Comment by quickwritereader** [comment]
Yes I was just checking to see if they initiated anything or not.

Probably, there is not any team left behind bountysource. As since feb-march they stopped replying to support emails or tickets.  And before, they were making promises.   
The main company behind is based on France.  
Bountysource was acquired several times. This one became the worst one.  I do not know what they are thinking or trying to do.
I never had such problem with bountysource before. the previous owners were sending stickers as an extra as well. And there were irc and slack chats.

**Comment by ghost** [comment]
> Yes I was just checking to see if they initiated anything or not.

Nothing from my side. Summer laziness, and I am not sure if my $900 cash-out is worth more effort (than writing up the issues here)

I guess that ideally, a California resident would file something re Bountysource, and someone in France re the mother company.

(IBM bounties are the most affected, both, cashed out and still in escrow)

**Comment by ghost** [comment]
@thesquash , app.bountysource is down, so I closed all my issues in favor of having a better position for:

- #1586

additionally, there is a footer-warning in the wiki now (added by a pinged former owner)

https://github.com/bountysource/core/wiki

**Comment by zhanglin2603** [comment]
Hey, I went over bountysource/core#1591 and can take this on (listed bounty: $155).
- My read: this is primarily a UI/UX task tied to "$139.50 cash out requested on the 19th of May; request not even acknowledged, let alone fu".
- I will focus on these issue-specific points first: cash, out, requested.
- First I will reproduce and confirm expected behavior for the UI/UX path.
- Then I will implement the fix with targeted tests and keep scope tight (good confidence).
- I will share a working PR update fast (24h for first patch, 48h for final polish) with verification notes.
If this approach works for you, I can start now.

**Comment by lmee** [comment]
I read "$139.50 cash out requested on the 19th of May; request not even acknowledged, let alone fulfilled, more than 7 weeks later" end-to-end and went through the full thread.

Problem I see: Over the past year, I earned $155 from three open bounties, all on the MATE Desktop project:.
Root-cause hypothesis: an FFI boundary mismatch (types/ownership/error handling) between wrapper code and native APIs.
Useful clue from comments: Hey, I went over bountysource/core 1591 and can take this on (listed bounty: $155). - My read: this is primarily a UI/UX task tied to "$139.50 cash out requested on the 19th of May.
First patch plan: define a minimal wrapper surface first, align types and ownership semantics with the native API, then add deterministic tests for correctness, memory safety, and failure-path behavior.
If this aligns with maintainer expectations, I will post repro evidence first and then open the scoped patch.
